src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
Well, my arg may not be the best, but we should think in reverse: what would be the argument for NOT putting the message here? @Taluu has two:
- the default message is not really use full
- issue with `\r`, `\n`, `*/`
These are good ones but I gave conter arguments: 
- recommend overwriting the default message
- forbid the special chars
Which imo means we have no moore arg for NOT putting the message here. Unless I missed something.